test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart default.snap.txt

```diff
@@ -41,22 +41,11 @@ wget
 redhat-lsb-core
 %end
 
-%post --nochroot
-exec < /dev/tty3 > /dev/tty3
-#changing to VT 3 so that we can see whats going on....
-/usr/bin/chvt 3
-(
-cp -va /etc/resolv.conf /mnt/sysimage/etc/resolv.conf
```

Comment:
I kind of wonder why this was where. Looks like we now no longer do it, but I'm wondering why it was needed in the first place.

Reply:
No idea, it should not be there. We should stick with default Anaconda behavior.